src/logistration/data/service.js

```diff
@@ -66,3 +66,23 @@ export async function getThirdPartyAuthContext(urlParams) {
     thirdPartyAuthContext: camelCaseObject(data),
   };
 }
+
+export async function getRegistrationForm() {
+  const requestConfig = {
+    headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+    isPublic: true,
+  };
+
+  const { data } = await getAuthenticatedHttpClient()
```

Comment:
can we use simple HTTPClient if the API is public?

Reply:
I dont know for sure... But how much difference it would make on performance.

Reply:
`getHttpClient()` is provided by @edx/frontend-platform/auth and can be used when the the API is public (as suggested in the header isPublic: True)

Reply:
I am not sure if using one over the other should have any performance impact